templates/compose/signoz.yaml

```
@@ -0,0 +1,624 @@
+                # This is added to ensure the uniqueness of the timeseries
+                # Otherwise, identical timeseries produced by multiple replicas of
+                # collectors result in incorrect APM metrics
+                - name: signoz.collector.id
+                - name: service.version
+                - name: browser.platform
+                - name: browser.mobile
+                - name: k8s.cluster.name
+                - name: k8s.node.name
+                - name: k8s.namespace.name
+                - name: host.name
+                - name: host.type
+                - name: container.name
+          extensions:
+            health_check:
+              endpoint: 0.0.0.0:13133
+            pprof:
+              endpoint: 0.0.0.0:1777
+          exporters:
+            clickhousetraces:
+              datasource: tcp://clickhouse:9000/signoz_traces
+              low_cardinal_exception_grouping: ${env:LOW_CARDINAL_EXCEPTION_GROUPING}
+              use_new_schema: true
+            signozclickhousemetrics:
+              dsn: tcp://clickhouse:9000/signoz_metrics
+            clickhouselogsexporter:
+              dsn: tcp://clickhouse:9000/signoz_logs
+              timeout: 10s
+              use_new_schema: true
+          service:
+            telemetry:
+              logs:
+                encoding: json
+            extensions:
+              - health_check
+              - pprof
+            pipelines:
+              traces:
+                receivers: [otlp]
+                processors: [signozspanmetrics/delta, batch]
+                exporters: [clickhousetraces]
+              metrics:
+                receivers: [otlp]
+                processors: [batch]
+                exporters: [signozclickhousemetrics]
+              metrics/prometheus:
+                receivers: [prometheus]
+                processors: [batch]
+                exporters: [signozclickhousemetrics]
+              logs:
+                receivers: [otlp]
+                processors: [batch]
+                exporters: [clickhouselogsexporter]
+      - type: bind
+        source: ./otel-collector-opamp-config.yaml
+        target: /etc/manager-config.yaml
+        content: |
+          server_endpoint: ws://signoz:4320/v1/opamp
+    environment:
+      - SERVICE_URL_OTELCOLLECTORHTTP_4318
+      - OTEL_RESOURCE_ATTRIBUTES=host.name=signoz-host,os.type=linux
+      - LOW_CARDINAL_EXCEPTION_GROUPING=false
+    healthcheck:
+      test: bash -c "exec 6<> /dev/tcp/localhost/13133"
+      interval: 30s
+      timeout: 5s
+      retries: 3
+
+  schema-migrator-sync:
+    image: signoz/signoz-schema-migrator:latest
+    command:
+      - sync
+      - --dsn=tcp://clickhouse:9000
+      - --up=
+    depends_on:
+      clickhouse:
+        condition: service_healthy
+    restart: on-failure
+    logging:
+      options:
+        max-size: 50m
+        max-file: "3"
+
+  schema-migrator-async:
+    image: signoz/signoz-schema-migrator:latest
```

Please pin to a static version rather than 'latest' to prevent the template from breaking due to upstream changes.

```suggestion
  schema-migrator-async:
    image: signoz/signoz-schema-migrator:v0.129.7
```